Documentation/install/system_requirements.rst

```diff
@@ -104,6 +104,41 @@ linked, either choice is valid.
    Users running Linux 4.10 or earlier with Cilium CIDR policies may face
    :ref:`cidr_limitations`.
 
+L7 proxy redirection currently uses ``TPROXY`` iptables actions as well
+as ``socket`` matches. For L7 redirection to work as intended kernel
+configuration must include the following modules:
+
+.. code:: bash
+
+        CONFIG_NETFILTER_XT_TARGET_TPROXY=m
+        CONFIG_NETFILTER_XT_MATCH_MARK=m
+        CONFIG_NETFILTER_XT_MATCH_SOCKET=m
+
+.. note::
+
+   As noted above, ``xt_socket`` kernel module is generally needed for
+   L7 redirection. Specifically, when ``xt_socket`` kernel module is
+   missing the forwarding of redirected L7 traffic does not work
+   in non-tunneled datapath modes. Since some notable kernels (e.g.,
+   COS) are shipping without ``xt_socket`` module, Cilium implements a
+   fallback compatibility mode to allow L7 policies and
+   visibility to be used with those kernels. Currently this fallback
+   disables ``ip_early_demux`` kernel feature in non-tunneled datapath
+   modes, which may decrease system networking performance.
```

Comment:
It's always tempting to call out specific parts of the documentation in `.. note`, but is this particular block of text important to any and every user going through the system requirements? Sure, it would be easier to miss if it's not called out in a `.. note`, but it also currently will affect very few people (COS is the only known distro that leads to this situation, and for GKE where COS is most popular, we're at least for now recommending to run in tunnel mode).

Reply:
removing the `.. note`